api/prometheus/v1/api.go

```diff
@@ -1139,8 +1139,12 @@ func (h *httpAPI) Series(ctx context.Context, matches []string, startTime, endTi
 		q.Add("match[]", m)
 	}
 
-	q.Set("start", formatTime(startTime))
-	q.Set("end", formatTime(endTime))
+	if !startTime.IsZero() {
```

Comment:
Can we add Series method comment to mention this behavior?

Reply:
I can, although the `Query` method (which already has this behavior) doesn't document it either. Do we still want to document this up (since this would be the case for all methods with a time.Time argument (exception being `Range` as it is required).